sql3/sql_complex_test.go

```diff
@@ -2631,3 +2638,216 @@ func TestPlanner_BulkInsert_FB1831(t *testing.T) {
 		t.Fatal("Expecting to be equal")
 	}
 }
+
+type tb struct {
+	Name  string
+	Type  arrow.DataType
+	Value interface{}
+}
+
+func simpleParquetMaker(f *os.File, numRows int64, input []tb) error {
+	mem := memory.NewGoAllocator()
+	chunks := make([]arrow.Array, 0)
+	for i := range input {
+		switch input[i].Type {
+
+		case arrow.PrimitiveTypes.Int64:
+			idbuild := array.NewInt64Builder(mem)
+			idbuild.AppendValues(input[i].Value.([]int64), nil)
+			newChunk := idbuild.NewArray()
+			chunks = append(chunks, newChunk)
+		case arrow.PrimitiveTypes.Float64:
+			fbuild := array.NewFloat64Builder(mem)
+			fbuild.AppendValues(input[i].Value.([]float64), nil)
+			newChunk := fbuild.NewArray()
+			chunks = append(chunks, newChunk)
+		case arrow.BinaryTypes.String:
+			sbuild := array.NewStringBuilder(mem)
+			sbuild.AppendValues(input[i].Value.([]string), nil)
+			newChunk := sbuild.NewArray()
+			chunks = append(chunks, newChunk)
+		}
+	}
+	// make schema
+	fields := make([]arrow.Field, len(input))
+	for i := range input {
+		fields[i].Name = input[i].Name
+		fields[i].Type = input[i].Type
+	}
+	schema := arrow.NewSchema(fields, nil)
+	rec := array.NewRecord(schema, chunks, numRows)
+	table := array.NewTableFromRecords(schema, []arrow.Record{rec})
+	props := parquet.NewWriterProperties(parquet.WithDictionaryDefault(false))
+	arrProps := pqarrow.DefaultWriterProps()
+	err := pqarrow.WriteTable(table, f, 4096, props, arrProps)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func TestPlanner_BulkInsertParquet_File(t *testing.T) {
+	// check that can pull parquet file from URL and load
+	c := test.MustRunCluster(t, 1)
+	defer c.Close()
+
+	_, _, err := sql_test.MustQueryRows(t, c.GetNode(0).Server, `create table j1 (_id ID, a INT, b DECIMAL(2), c STRING);`)
+	if err != nil {
```

Comment:
I would prefer we use testify `assert.NoError(t, err)` (along with any other testify methods) in tests going forward.

Reply:
forgot all about testify, will update